game-engine/src/commonMain/kotlin/za/co/entelect/challenge/game/engine/renderer/WormsGameDetails.kt

```diff
@@ -30,4 +32,15 @@ class WormsGameDetails(config: GameConfig, wormsMap: WormsMap, player: WormsPlay
         return arrayMap.map { PrintableMapCell.buildForPerspectivePlayer(it, player) }
     }
 
+    private fun getOpponentsLastCommand(wormsMap: WormsMap, player: WormsPlayer): String {
+        val opponentFeedback = wormsMap
+            .getFeedback(currentRound - 1)
+            .filter { it.playerId != player.id }
+        return if (opponentFeedback.size == 0) {
+            "nothing"
```

Comment:
Try this one 😉
`CommandStrings.NOTHING.string`

Reply:
Duh!  Thanks :)